content/research/index.html

```diff
@@ -48,6 +48,9 @@ <h3>JNation 2024, Coimbra</h3>
   <a class="extern" href="https://jnation.pt/">JNation</a>,
   2024/06/04
 </p>
+<ul>
+  <li><a class="extern" href="https://youtu.be/Yqtj0XoikmQ?t=24572">Video</a></li>

```

Comment:
This link is to the entire conference track, lets use the one that is dedicated to the talk.

```suggestion
  <li><a class="extern" href="https://www.youtube.com/watch?v=7r8a12oYgEU">Video</a></li>

```

Reply:
```suggestion
  <li><a class="extern" href="https://youtu.be/7r8a12oYgEU">Video</a></li>
```